spec/cc/analyzer/issue_validations/content_validation_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "spec_helper"
+
+module CC::Analyzer::IssueValidations
+  describe ContentValidation do
+    describe "#valid?" do
+      it "is valid without any content" do
+        expect(ContentValidation.new({})).to be_valid
+      end
+
+      it "is valid with content with body" do
+        expect(ContentValidation.new("content" => { "body" => "hi"})).to be_valid
+      end
+
+      it "is not valid with content as string" do
+        expect(ContentValidation.new("content" => "hi")).not_to be_valid
+      end
+
+      it "is not valid with content as nil" do
+        expect(ContentValidation.new("content" => nil)).not_to be_valid
```

Comment:
Honestly not sure. It doesn't comply with the spec, though, so may as well catch it?